src/hb-ot-color-sbix-table.hh

```diff
@@ -337,12 +337,13 @@ struct sbix
                    const void *dst_base,
                    unsigned int i,
                    unsigned int sbix_len) const {
+    // Push first so reverting doesn't fail.
+    c->serializer->push ();
```

Comment:
Guess better to be called in the caller, serialize_strike_offsets, so number of push and pop there can be balanced?

Reply:
Good idea. Done.